index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;600&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./src/style.css">
+    <title>Apple</title>
+</head>
+
+<body>
+    <header>
+        <nav>
+            <div><i class="fab fa-apple"></i></div>
+            <div>Store</div>
+            <div>Mac</div>
+            <div>iPod</div>
+            <div>iPhone</div>
+            <div>iPad</div>
+            <div>iTunes</div>
+            <div>Support</div>
+            <div>
+                <form>
+                    <button type="submit"><i class="fas fa-search"></i></button>
+                    <input />
```

Comment:
- [ ] Use the correct [input type ](https://www.w3schools.com/html/html_form_input_types.asp)